### PR TITLE
Move VideoFrameMetadata dfn to WebCodecs

### PR DESCRIPTION
**Pull request by tidoust**
A registry cannot contain normative IDL, so the base definition of the `VideoFrameMetadata` dictionary should remain in WebCodecs.

Also note that while the registry will be published as a "W3C Draft Registry", the document in the repo remains an "Editor's Draft". Updating the status accordingly.

***
<a href="https://pr-preview.s3.amazonaws.com/tidoust/webcodecs/pull/596.html" title="Last updated on Oct 21, 2022, 12:18 PM UTC (faef414)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/webcodecs/596/d7edc37...tidoust:faef414.html" title="Last updated on Oct 21, 2022, 12:18 PM UTC (faef414)">Diff</a>

### CODE REVIEW

File: video_frame_metadata_registry.src.html [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <pre class='metadata'>
 Title: WebCodecs VideoFrame Metadata Registry
 Repository: w3c/webcodecs
-Status: DRY
+Status: ED
```
Comment (youennf) [inline video_frame_metadata_registry.src.html:4]: Why this change?

**Review by youennf — COMMENTED** [review]
LGTM

File: video_frame_metadata_registry.src.html (hunk 1 above)
Comment (tidoust) [reply video_frame_metadata_registry.src.html:4]: Although it has "Draft" in the title, `DRY` is a status for documents published under /TR. All documents in the repo are "Editor's Drafts". There is no `DRY-ED` status in authoring tools for now to qualify an "Editor's Draft of a Registry", so using "ED" as fallback (I see the status of the Editor's Draft of the Codec registry is `NOTE-ED` for now, that's really treated the same way as `ED` internally and produces the same boilerplate).

**Review by youennf — APPROVED** [review]

**Comment by dalecurtis** [comment]
Hmm, https://github.com/w3c/webcodecs/actions/runs/3298886029 failed after this. Not sure what's going on. @tidoust ?

**Comment by tidoust** [comment]
There seems to have been a temporary network hiccup when exchanging with the W3C Echidna server to check on progress of a publication request, but I confirm everything got published!
